getting_started.md

```diff
@@ -78,6 +78,20 @@ https://github.com/dwyl/auth_plug#2-get-your-auth_api_key-
 
 ## Setup
 
+### On macOS? Use our management script
+
+1. Run the following command to download and run the Python3 script
+
+```
+bash <(curl -sL https://git.io/JUIen)
```

Comment:
Links to: https://raw.githubusercontent.com/dwyl/smart-home-security-system/install/go.sh `404`:
![image](https://user-images.githubusercontent.com/194400/161087090-2c31edd1-9fac-418d-9056-e9547ad663d9.png)